api/v1/helpers/verbs/utils.js

```diff
@@ -150,28 +150,67 @@ function buildFieldList(params) {
 /**
  * Checks if the model instance is writable by a user. The username is extracted
  * from the header if present, if not the user name of the logged in user is
- * used
+ * used. This always resolves to the model instance if isEnabled is set to
+ * false.
+ * @param {Object} req  - The request object
+ * @param {Object}  modelInst - DB Model instance
+ * @param {isEnabled} isEnabled - A flag, when set, lets the caller of this
```

Comment:
Does every caller need to pass in the "isEnabled" state, or can this function just look it up here?